### PR TITLE
Release 0.14.0

### PR DESCRIPTION
**Pull request by zimlu02**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+* Fix syntax analysis termination with unexpected copy statements GH-464 ([6b0ad8e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/6b0ad8e5eabbd4f97edaa625184358255ae3e1f6)), closes [#464](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/464)
+
+### Features
+* Outline view support [#481](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/481) ([16ba495](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/16ba495c5226575a2d530a3c1bc9a9dec156d2d5))
+* Provide default COBOL code snippets [#484](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/484) ([c98df71](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/c98df710f945cc25e849563e2f89a864db8d06c7))
+* Underscore the copy statement if it contains a syntax error [#450](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/450) ([082bbf7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/082bbf7765158ea4bc54ac31b41972c0938dbcc4))
+
+### Performance Improvements
+* Remove unnecessary token operations on replacing ([0a59352](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/0a593526182e9259dbb30e6a329363fe9686e8e0))
+
```
Comment (temanbrcom) [inline CHANGELOG.md:20]: Do we need to mention the performance improvements in the changelog? It seems to be technical details

**Comment by grianbrcom** [comment]
It looks OK.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+### Bug Fixes
+* Allow multi-line errors underscoring GH-486 ([b79bd82](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/b79bd82a1446e22316d3b4aec03e1cd9b1159a88)), closes [#486](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/486)
+* Fix a typo in DIVIDE-REMAINDER code snippets ([00cd27d](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/00cd27d1e63369d6695be05654bad7fdca4aa7a7))
+* Fix checking program ID with quotes [#477](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/477) ([2c247ff](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/2c247fff1766262a3651a8fa9a4e587b54af3921))
+* Fix incorrect indicator area error message [#494](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/494) ([5a6501e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/5a6501ead0feffb4038ce67424b04d2e28b82c21))
+* Fix syntax analysis termination with unexpected copy statements GH-464 ([6b0ad8e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/6b0ad8e5eabbd4f97edaa625184358255ae3e1f6)), closes [#464](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/464)
```
Comment (temanbrcom) [inline CHANGELOG.md:11]: Previously we didn't add the links, should we do it now?

**Review by grianbrcom — APPROVED** [review]
:seedling:

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+* Fix syntax analysis termination with unexpected copy statements GH-464 ([6b0ad8e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/6b0ad8e5eabbd4f97edaa625184358255ae3e1f6)), closes [#464](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/464)
+
+### Features
+* Outline view support [#481](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/481) ([16ba495](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/16ba495c5226575a2d530a3c1bc9a9dec156d2d5))
+* Provide default COBOL code snippets [#484](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/484) ([c98df71](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/c98df710f945cc25e849563e2f89a864db8d06c7))
+* Underscore the copy statement if it contains a syntax error [#450](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/450) ([082bbf7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/082bbf7765158ea4bc54ac31b41972c0938dbcc4))
+
+### Performance Improvements
+* Remove unnecessary token operations on replacing ([0a59352](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/0a593526182e9259dbb30e6a329363fe9686e8e0))
+
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:20]: I think here we should only extract the changes that are significant for the user

**Review by temanbrcom — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (zimlu02) [reply CHANGELOG.md:11]: why not?

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (zimlu02) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:20]: like having full list of changes available for the contributors and a short one for the users?

File: CHANGELOG.md (hunk 2 above)
Comment (temanbrcom) [reply CHANGELOG.md:11]: It's up to you, it is just a question of the changelog consistency

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:20]: Yes, we had a discussion about this some time ago. Just have a look at the previous release - we have the changelog with the pattern Added/Changed/Fixed without any technical details

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,22 @@
...
+- Breadcrumb support
+- Provide default COBOL code snippets
+- Underscore the copy statement if it contains a syntax error
+
+#### Fixed
+- Allow multi-line errors underscoring GH-486
+- Fix a typo in DIVIDE-REMAINDER code snippets
+- Fix checking program ID with quotes
+- Fix incorrect indicator area error message
+- Fix syntax analysis termination with unexpected copy statements GH-464
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:18]: In this context, the list elements should begin with a noun

Comment (temanbrcom) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:18]: - Outline view support
- Breadcrumb support
- Default COBOL code snippets
- Underscoring the copy statement if it contains a syntax error

#### Fixed
- Multi-line errors underscoring GH-486
- A typo in DIVIDE-REMAINDER code snippets
- Program ID check with quotes
- Incorrect indicator area error message
- Syntax analysis termination with unexpected copy statements GH-464

Comment (zimlu02) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md]: omg - nice exercise :-D

**Review by temanbrcom — APPROVED** [review]
🥇

**Pull request merged**